Update _model.py

bjornekstrom opened the pull request:
Edited `get_feature_names()` to `get_feature_names_out()` for noun passing to Concept to work with newer versions of scikit-learn.

bjornekstrom committed:
Update _model.py

Edited get_feature_names() to get_feature_names_out() for noun passing to Concept to work with newer versions of scikit-learn.

MaartenGr commented:
Thank you for the PR! Could you also make sure that the minimum version of scikit-learn is updated to reflect this? This will give an error if you are using older versions of scikit-learn. In BERTopic, for example, I made sure that both older and newer versions of scikit-learn could be used. That would be my preferred method here:

https://github.com/MaartenGr/BERTopic/blob/a6fefe4283e82f23482ac46164ac75409bd7f5dc/bertopic/_bertopic.py#L1095-L1100

bjornekstrom committed:
Update _model.py

Enabled `get_feature_names` to work with both old and new versions of scikit-learn.

MaartenGr commented:
Awesome, LGTM!
